man/userdbctl.xml

```diff
@@ -59,7 +59,13 @@
         user friendly, human readable output is generated; if <literal>table</literal> a minimal, tabular
         output is generated; if <literal>json</literal> a JSON formatted output is generated. Defaults to
         <literal>friendly</literal> if a user/group is specified on the command line,
-        <literal>table</literal> otherwise.</para></listitem>
+        <literal>table</literal> otherwise.</para>
+
+        <para>Note that most output formats do not show all available information. In particular,
+        <literal>classic</literal> and <literal>table</literal> show only the most important fields. Various
+        modes also do not show password strings. Use <literal>json</literal> to view all fields, including
+        any authentication secrets.</para>
```

Comment:
the "secrets" are not part of the record, only the hash version of the secrets are, and they are not really secret. I mean, they kinda are on traditional Linux because /etc/shadow is root-only accessible, but that's mostly snake-oil, if the hash function would work making it read-only wouldn't really be necessary, except maybe adding yet another level of security.

Hence maybe say "any authentication secrets" → "any authentication fields"? or so?